### ISSUE TITLE
Question from KDD 2020 Presentation

### ISSUE THREAD

**Issue by jbohning** [shap/shap#1384]
Is feature selection for EBMs as important as with explainable models (linear, logistic)? Or is it less important like with black box models (like RF, GBM, neural nets). How do we handle feature selection for EBMS? 

Also, is there any chance this package will be available soon for r?

**Comment by slundberg** [comment]
These are questions best asked on the interpretml/interpret github, but in general EBMs are GBMs with a specific focus on building very shallow trees and minimizing interactions. So whatever intuitions you have about an XGB model with column and instance subsampling and a low learning rate and a shallow tree depth should carry over to EBMs.
